drivers/Makefile.dep

```diff
@@ -32,6 +32,21 @@ ifneq (,$(filter cc110x,$(USEMODULE)))
   endif
 endif
 
+ifneq (,$(filter cc2420,$(USEMODULE)))
+  USEMODULE += cc2420
```

Comment:
The reason for the at86rf2xx USEMODULE check is that the driver is checking
for a wildcard and then adding the generic driver if it matches. The
at86rf2xx driver supports the rf231, rf233, rf212b, (possibly more), and is
written in a way that it relies on USEMODULE to tell it which hardware it
should expect.
Den 5 jul 2016 12:33 skrev "kYc0o" <notifications@github.com>:

In drivers/Makefile.dep
<https://github.com/RIOT-OS/RIOT/pull/5591#discussion_r69541037>:

> @@ -32,6 +32,21 @@ ifneq (,$(filter cc110x,$(USEMODULE)))
>    endif
>  endif
>
> +ifneq (,$(filter cc2420,$(USEMODULE)))
> +  USEMODULE += cc2420

Oh yeah, it was also used with at86rf2xx so I added it, but you're right,
it's not required. Should I remove it?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/RIOT-OS/RIOT/pull/5591/files/ffd03d77950ecaa2210bc2efb066a12ad9728cfd#r69541037>,
or mute the thread
<https://github.com/notifications/unsubscribe/AATYQl9hCQ81LrCAtqWHIs1iLu4Rowqzks5qSjL7gaJpZM4JDOnR>
.